public/assets/.sprockets-manifest-413d41b33cf339d3ef4b4c35f32e8402.json

```diff
@@ -0,0 +1 @@
igest":"d6b8be8bae4f838685f2f88e112702f4fef0e9c169d4ca25d52bcb14fdf7bb3b","integrity":"sha256-1ri+i65Pg4aF8viOEScC9P7w6cFp1Mol1SvLFP33uzs="},"edit-72a5ccd30f62c136fe038aea011169f6503d7f2799d2874fda91f541642bf664.png":{"logical_path":"edit.png","mtime":"2018-04-12T15:20:10+03:00","size":21973,"digest":"72a5ccd30f62c136fe038aea011169f6503d7f2799d2874fda91f541642bf664","integrity":"sha256-cqXM0w9iwTb+A4rqARFp9lA9fyeZ0odP2pH1QWQr9mQ="},"hand-d0394b3f15228194d1f7338a314f909175699c021f48ae2912b6c4eee45ae48a.png":{"logical_path":"hand.png","mtime":"2018-04-12T16:39:15+03:00","size":4765,"digest":"d0394b3f15228194d1f7338a314f909175699c021f48ae2912b6c4eee45ae48a","integrity":"sha256-0DlLPxUigZTR9zOKMU+QkXVpnAIfSK4pErbE7uRa5Io="},"ok-bc1d6c587bbe12cd805c455b607075d14d7dbd70b02fd24bf2ceddfd447d575d.png":{"logical_path":"ok.png","mtime":"2018-04-12T15:20:10+03:00","size":26682,"digest":"bc1d6c587bbe12cd805c455b607075d14d7dbd70b02fd24bf2ceddfd447d575d","integrity":"sha256-vB1sWHu+Es2AXEVbYHB10U19vXCwL9JL8s7d/UR9V10="},"prodobro1-7eb2a1e25ade1b4785a7a1d98eb72810bb493acae4735d7b2d75dae9ff7331b0.png":{"logical_path":"prodobro1.png","mtime":"2018-04-12T15:20:10+03:00","size":80989,"digest":"7eb2a1e25ade1b4785a7a1d98eb72810bb493acae4735d7b2d75dae9ff7331b0","integrity":"sha256-frKh4lreG0eFp6HZjrcoELtJOsrkc117LXXa6f9zMbA="},"prodobro2-9d9dcdf002f446dd3b77ccea0091ee1f75713ad69bdc8288e23a41834dfe29c2.png":{"logical_path":"prodobro2.png","mtime":"2018-04-12T16:39:15+03:00","size":6625,"digest":"9d9dcdf002f446dd3b77ccea0091ee1f75713ad69bdc8288e23a41834dfe29c2","integrity":"sha256-nZ3N8AL0Rt07d8zqAJHuH3VxOtab3IKI4jpBg03+KcI="},"application-875cca143a022ebb6768c8901324671170a0eaceaa9e3d4969cb2f4f71e3e2b7.js":{"logical_path":"application.js","mtime":"2018-04-16T15:43:52+03:00","size":217110,"digest":"875cca143a022ebb6768c8901324671170a0eaceaa9e3d4969cb2f4f71e3e2b7","integrity":"sha256-h1zKFDoCLrtnaMiQEyRnEXCg6s6qnj1JacsvT3Hj4rc="},"application-b1b11a83199bd0be2335bfd4ac6c28ce9b7a64da1ec0d2af3b73b67fceeead78.css":{"logical_path":"application.css","mtime":"2018-04-16T15:59:06+03:00","size":126325,"digest":"b1b11a83199bd0be2335bfd4ac6c28ce9b7a64da1ec0d2af3b73b67fceeead78","integrity":"sha256-sbEagxmb0L4jNb/UrGwozpt6ZNoewNKvO3O2f87urXg="}},"assets":{"email.css":"email-d6b8be8bae4f838685f2f88e112702f4fef0e9c169d4ca25d52bcb14fdf7bb3b.css","edit.png":"edit-72a5ccd30f62c136fe038aea011169f6503d7f2799d2874fda91f541642bf664.png","hand.png":"hand-d0394b3f15228194d1f7338a314f909175699c021f48ae2912b6c4eee45ae48a.png","ok.png":"ok-bc1d6c587bbe12cd805c455b607075d14d7dbd70b02fd24bf2ceddfd447d575d.png","prodobro1.png":"prodobro1-7eb2a1e25ade1b4785a7a1d98eb72810bb493acae4735d7b2d75dae9ff7331b0.png","prodobro2.png":"prodobro2-9d9dcdf002f446dd3b77ccea0091ee1f75713ad69bdc8288e23a41834dfe29c2.png","application.js":"application-875cca143a022ebb6768c8901324671170a0eaceaa9e3d4969cb2f4f71e3e2b7.js","application.css":"application-b1b11a83199bd0be2335bfd4ac6c28ce9b7a64da1ec0d2af3b73b67fceeead78.css"}}
```

Comment:
`public/assets` folder should be in` .gitignor`
Assets should be compiled during the deploy process